modules/govuk/manifests/apps/content_audit_tool/db.pp

```diff
@@ -0,0 +1,23 @@
+# == Class: govuk::apps::content_audit_tool::db
+#
+# === Parameters
+#
+# [*password*]
+#   The DB instance password.
+#
+# [*backend_ip_range*]
+#   Backend IP addresses to allow access to the database.
+#
```

Comment:
The rds parameter is missing from the list in the comment.

Reply:
Addressed in https://github.com/alphagov/govuk-puppet/pull/7141/commits/9c4dedacec13d8f0f2fbdf3dd1f0620a84e4c920